jbpm-human-task/jbpm-human-task-core/src/test/java/org/jbpm/services/task/assignment/LifeCycleLocalCommandBasedTest.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2015 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+package org.jbpm.services.task.assignment;
+
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.Persistence;
+
+import org.jbpm.services.task.HumanTaskServiceFactory;
+import org.jbpm.services.task.LifeCycleBaseTest;
+import org.junit.After;
+import org.junit.Before;
+import org.kie.internal.task.api.InternalTaskService;
+
+import bitronix.tm.resource.jdbc.PoolingDataSource;
+
+
+
+public class LifeCycleLocalCommandBasedTest extends LifeCycleBaseTest {
```

Comment:
What is the purpose of having this test class here? It seems like it has been copied from `org.jbpm.services.task.LifeCycleLocalCommandBasedTest` without any change.